provider/spark.go

```diff
@@ -322,53 +311,34 @@ func (db *DatabricksExecutor) RunSparkJob(args *[]string, store FileStore) error
 	// if err := clusterClient.Edit(setConfigReq); err != nil {
 	// 	return fmt.Errorf("Could not modify cluster to accept spark configs; %v", err)
 	// }
-	jobsClient := db.client.Jobs()
-	pythonTask := azureModels.SparkPythonTask{
+	pythonTask := jobs.SparkPythonTask{
 		PythonFile: db.PythonFileURI(store),
 		Parameters: args,
 	}
-	type CustomCreateReq struct {
-		ExistingCluster string                       `json:"existing_cluster_id,omitempty" url:"existing_cluster_id,omitempty"`
-		SparkPythonTask *azureModels.SparkPythonTask `json:"spark_python_task,omitempty" url:"spark_python_task,omitempty"`
-		Name            string                       `json:"name,omitempty" url:"name,omitempty"`
-	}
-	createJobRequest := CustomCreateReq{
-		ExistingCluster: db.cluster,
-		SparkPythonTask: &pythonTask,
-		Name:            "Databricks spark submit job 2",
-	}
-	jsonResp, err := databricks.PerformQuery(jobsClient.Client.Option, http.MethodPost, "/jobs/create", createJobRequest, nil)
-	if err != nil {
-		return fmt.Errorf("could not create job: %w", err)
-	}
-	var resp azureHTTPModels.CreateResp
-	err = json.Unmarshal(jsonResp, &resp)
+	ctx := context.Background()
+	id := uuid.New().String()
+
+	jobToRun, err := db.client.Jobs.Create(ctx, jobs.CreateJob{
+		Name: fmt.Sprintf("featureform-job-%s", id),
```

Comment:
In the future for all spark, Id like to change the job name to reflect what resource is being created